apps/docs/content/components/Avatar/index.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Avatar
+description: Use Avatar to display a thumbnail representation of a person.
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+![An image showing two different avatars, a circular and a square one.](https://user-images.githubusercontent.com/175638/234820833-c803cb11-2bc3-4597-88f3-2cf630a46702.png)
+
+## Usage
+
+Use avatars to display a thumbnail that represents individuals in various contexts, such as testimonials, speaker sessions, blog post authors, and beyond.
+
+## Options
+
+### Size
+
+Avatar offers various sizes to fit different layout and visual scale needs. Sizes range from 32 to 80 pixels. Ensure the avatar is proportionate to its accompanying text.
```

Comment:
Would be cool to see an example of a do/don't for this: 

> Ensure the avatar is proportionate to its accompanying text

These are great opportunities to show the example visually, to emphasise on the importance. Also just curious what a do/don't would look like here 😀